ILRepack.IntegrationTests/NuGet/TestsPlatformEnglightenmentProvider.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Reactive.PlatformServices;
+
+namespace ILRepack.IntegrationTests.NuGet
+{
+    public class TestsPlatformEnglightenmentProvider : CurrentPlatformEnlightenmentProvider
```

Comment:
This is needed in order to have the proper exception thrown. Otherwise we don't know why it fails during an observable sequence :(